src/locales/pl/registerForm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "registerTitle": ["Utwórz konto", "REJESTRACJA"],
```

Comment:
let's create a `title` and `shortTitle` instead of array of tittles

Reply:
should be fixed in 9e244cb